### PR TITLE
Location field moved to settings page

### PR DESCRIPTION
**Pull request by SaturnusDJ**

### CODE REVIEW

File: desktop-app/js/tinder++.swipe.js [hunk 1]
```diff
@@ -6,14 +6,9 @@
   module.controller('SwipeController', function SwipeController($scope, $http, $timeout, $interval, API) {
```
Comment (SaturnusDJ) [inline desktop-app/js/tinder++.swipe.js]: `$http` doesn't seem to be used. Consider remove.

**Comment by mayeaux** [comment]
Perfect, will check this out tomorrow but this is exactly what a PR should look like, thanks! :D

File: desktop-app/index.html [hunk 2]
```diff
@@ -237,6 +236,18 @@ <h3 class="fader" style="display: inline;">{{user.distance_mi}} miles away, Acti
     <h1>Settings</h1>
     <form name="tppSettings">
       <fieldset>
+        <legend>Location</legend>
+			  <p>
+				<a href="" ng-click="toggleLocation()" >{{getCookie('currentCity') ? getCookie('currentCity') : 'set location'}}</a>
```
Comment (stuwil) [inline desktop-app/index.html]: PR looks good! Would you mind capitalizing to 'Set Location' to match the other settings?

File: desktop-app/js/tinder++.swipe.js (hunk 1 above)
Comment (stuwil) [inline desktop-app/js/tinder++.swipe.js]: Thanks, go ahead and remove - this looks ready to merge after that.

Comment (SaturnusDJ) [inline desktop-app/js/tinder++.swipe.js]: Done!!

**Pull request merged**